### PR TITLE
More detailed hints in case of error

### PR DESCRIPTION
**Pull request by t-zimmermann**

### CODE REVIEW

**Comment by t-zimmermann** [comment]
Hi, I'm pretty new to this github-things.....
When I proposed this change about two weeks ago, I got the answer from one of the maintainers that I shouldn't worry about that, because they made some failures in their tests. Now this useful pull request still is in the line.... Could anybody please reply on this? Thanks!

**Comment by Shade634** [comment]
Grammar is quite important since it can lead to wrong interpretation. I however do agree that additional information is just as important.

File: Resources/Private/Partials/Errors/BackendLayoutNotSupported.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h4 class="alert-title">
 					You previously selected "{layout}" as backend layout, which is not supported by the currently selected theme.
 				</p>
 				<p>
-					Please select another one in the properties of your page.
+					Please select another one in the properties of your page. If you created your own backendlayout and you are sure to have selected the right one, please verify that the identifier in your backend-configuration reads exactly (capitalization inclusive!) like the filename of your theme-template.
```
Comment (Shade634) [inline Resources/Private/Partials/Errors/BackendLayoutNotSupported.html]: If you use your own backend-layout, please make sure that you have selected the correct one. 
Verify that the identifier in your backend-configuration matches the filename of your theme-template. (case sensitive).

**Comment by kaystrobach** [comment]
thank you i combined your hint with the notes of @Shade634 and so it got merged

**Pull request merged**